documentation/book/proc-accessing-kafka-using-ingress.adoc

```diff
@@ -50,13 +50,13 @@ spec:
 
 . Create or update the resource.
 +
-[source,shell,subs=+quotes]
-oc apply -f _your-file_
+[source,shell,subs="+quotes,attributes+"]
+kubectl apply -f _your-file_
```

Comment:
Not {cmdcli} here?

Reply:
It's in a module that's only included when kube doc is being generated, but there's no harm in fixing.